plugins/playlist/src/components/PlaylistEditDialog/PlaylistEditDialog.tsx

```diff
@@ -74,13 +75,50 @@ export const PlaylistEditDialog = ({
 }: PlaylistEditDialogProps) => {
   const classes = useStyles();
   const identityApi = useApi(identityApiRef);
+  const playlistApi = useApi(playlistApiRef);
+  const playListApiData = playlistApi.getAllPlaylists({ editable: true });
+
+  const [editingOtherFields, setEditingOtherFields] = useState(false);
+
+  const fetchAndProcessData = async () => {
```

Comment:
Not really processing anything, this function should be removed.

To be clear, calling `await playListApiData` here only waits for the one, single, request that was made above. It never re-fetches anything.